docs-partials/builder/vsphere/iso/vAppConfig-not-required.mdx

```diff
@@ -0,0 +1,11 @@
+<!-- Code generated from the comments of the vAppConfig struct in builder/vsphere/iso/step_create.go; DO NOT EDIT MANUALLY -->
+
+- `properties` (map[string]string) - Set values for the available vApp Properties to supply configuration parameters to a virtual machine cloned from
```

Comment:
Regarding Lucas comment about the properties field being a map[string]string I will defer to your expertise on vSphere and assume that it is not an issue. But please let me know if it is something that we need to call out in the docs further. 

Also does "These generally come from an existing template that was created from an imported OVF or OVA file." apply to the ISO builder?

Reply:
i'll need to check, but I don't think it directly applies. I also agree that moving it to common would mean more ideal.

I'm happy to hold this one for further evaluation and testing to be honest. I only addressed the failing CI.

Reply:
I started to take a look at the PR this evening and the descriptions need some updates since it's a copy/paste from the comments in the clone builder for the clone step. I'll make these changes accordingly.

I need to chat with @dilyar85 to see if this should be in common, but I don't think it will apply to the supervisor builder and therefore not be in common.

Reply:
This doesn't apply to the current supervisor builder. To set a VM's vApp properties in Supervisor, users need to specify the `bootstrap_provider` as "vAppConfig" and provide the properties in the `bootstrap_data_file`. This is required by the [VM-Operator v1alpha1 API](https://vm-operator.readthedocs.io/en/latest/concepts/workloads/guest/#vappconfig) that the supervisor builder uses internally.

In VM-Operator v1alpha2 (in the upcoming vSphere 8 Update 3 release), there is a new `properties` field introduced in the [VirtualMachineBootstrapVAppConfigSpec](https://vm-operator.readthedocs.io/en/latest/ref/api/v1alpha2/#virtualmachinebootstrapvappconfigspec). So making this common could be helpful as the supervisor builder would be able to set that field from this common config. But we would likely introduce a new config when we switch the supervisor builder to v1a2, since that field also supports setting value from K8s secret key selector.

I am fine with leaving this as is and making it common if needed when we add v1a2 support in the supervisor builder.